File: catchcraycat/src/main/java/com/encdata/catchcraycat/MainActivity.java
package com.encdata.catchcraycat;

import android.app.Activity;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends Activity {

    private int COL = 10;
    private int ROW = 10;

    Dot matrix[][] = new Dot[COL][ROW];

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
//        setContentView(R.layout.activity_main);
        setContentView(new Playground(this));
    }
}

File: kotlin_demo/src/main/java/com/encdata/kotlin/MainActivity.kt
package com.encdata.kotlin

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        process()
    }

    private fun process() {
        println(sum(1, 2))
    }

    private fun sum(a: Int, b: Int): Int {   // Int 参数，返回值 Int
        return a + b
    }
}
